deployments/data-hub/data-hub--staging.yaml

```diff
@@ -0,0 +1,230 @@
+
+        # scheduler interval
+          # At 07:00 on every day-of-week from Monday through Friday :
+        - name: GMAIL_DATA_PIPELINE_SCHEDULE_INTERVAL
+          value: "0 7 * * 1-5"
+        - name: CROSS_REF_IMPORT_SCHEDULE_INTERVAL
+          value: "@daily"
+        - name: GOOGLE_SPREADSHEET_SCHEDULE_INTERVAL
+          value: "@daily"
+          # At 02:25 :
+        - name: WEB_API_SCHEDULE_INTERVAL
+          value: "25 2 * * *"
+          # At minute 10 past every 2nd hour :
+        - name: S3_CSV_SCHEDULE_INTERVAL
+          value: "10 */2 * * *"
+          # At minute 50 past every 3rd hour from 8 through 18 :
+        - name: MONITOR_DATA_HUB_PIPELINE_HEALTH_SCHEDULE_INTERVAL
+          value: "50 8-18/3 * * *"
+        - name: MATERIALIZE_BIGQUERY_VIEWS_SCHEDULE_INTERVAL
+          value: "@hourly"
+        - name: SCHEDULE_INTERVAL
+          value: "@weekly"
+        - name: DATA_SCIENCE_FORECAST_SCHEDULE_INTERVAL
+          value: "@hourly"
+          # At minute 40 past every 3rd hour :
+        - name: DATA_SCIENCE_PEERSCOUT_RECOMMEND_SCHEDULE_INTERVAL
+          value: "40 */3 * * *"
+        - name: DATA_SCIENCE_PEERSCOUT_EDITOR_PUBMED_SCHEDULE_INTERVAL
+          value: "@hourly"
+          # At every 20th minute :
+        - name: EJP_XML_SCHEDULE_INTERVAL
+          value: "*/20 * * * *"
+          # At minute 10 past every 2nd hour :
+        - name: EXTRACT_KEYWORDS_SCHEDULE_INTERVAL
+          value: "10 */2 * * *"
+        - name: SURVEYMONKEY_DATA_PIPELINE_SCHEDULE_INTERVAL
+          value: "@monthly"
+
+        # general:
+        - name: DEPLOYMENT_ENV
+          value: staging
+        - name: AIRFLOW_APPLICATIONS_DIRECTORY_PATH
+          value: /opt/airflow/auxiliary_data_pipeline_files
+        - name: GOOGLE_APPLICATION_CREDENTIALS
+          value: /dag_secret_files/gcloud/credentials.json
+        - name: GMAIL_ACCOUNT_SECRET_FILE
+          value: /dag_secret_files/gmail/gmail_credentials.json
+        - name: SURVEYMONKEY_SECRET_FILE
+          value: /dag_secret_files/surveymonkey/surveymonkey_credentials.json
+        - name: IS_GMAIL_END2END_TEST
+          value: "False"
+        - name: INITIAL_S3_FILE_LAST_MODIFIED_DATE
+          value: "2020-02-14 13:00:00"
+        - name: MATERIALIZE_BIGQUERY_VIEWS_GCP_PROJECT
+          value: "elife-data-pipeline"
+        - name: MATERIALIZE_BIGQUERY_VIEWS_DATASET
+          value: "staging"
+        - name: INITIAL_S3_XML_FILE_LAST_MODIFIED_DATE
+          value: "2012-01-01 00:00:00"
+        - name: HEALTH_CHECK_URL
+          valueFrom:
+            secretKeyRef:
+              name: monitoring-urls
+              key: health_check_url
+        - name: DATA_HUB_MONITORING_SLACK_WEBHOOK_URL
+          valueFrom:
+            secretKeyRef:
+              name: monitoring-urls
+              key: slack_webhhok_url
+      sync:
+        resources:
+          requests:
+            memory: 400Mi
+            cpu: 800m
```

Comment:
Sometimes these values are quoted, other times not. Not a major issue but nice to be consistent. Something to followup on later if you fancy.

Reply:
Yup, depends if I wrote it or copy pasted. Will standardise on unquoted, as that's both my preference and follows other deployment files.